sql-server-2014-alwayson-existing-vnet-and-ad/Readme.md

```diff
@@ -0,0 +1,107 @@
+# Create a SQL Server 2014 Always On Availability Group in an existing Azure VNET and an existing Active Directory instance
```

Comment:
Is this template the same as the existing sql always on but without vnet and ad?